src/sentry/notifications/types.py

```diff
@@ -33,6 +33,7 @@ class NotificationSettingEnum(ValueEqualityEnum):
     SPIKE_PROTECTION = "spikeProtection"
     MISSING_MEMBERS = "missingMembers"
     REPORTS = "reports"
+    CRON_MONITORS = "cronMonitors"
```

Comment:
Is it going to be confusing to customers that cron monitor notifications are JUST the broken notices and NOT the issues that get created?

Reply:
dang yea you're right, lemme change the name to BROKEN_MONITORS, and update the help text to be very explicit here